src/h3lib/include/algos.h

```diff
@@ -40,4 +40,10 @@ void h3SetToVertexGraph(const H3Index* h3Set, const int numHexes,
 // Create a LinkedGeoPolygon from a vertex graph
 void _vertexGraphToLinkedGeo(VertexGraph* graph, LinkedGeoPolygon* out);
 
+// Internal functions for polyfill
+int _get_edge_hexagons(const Geofence* geofence, int numHexagons, int res,
```

Comment:
Would it make more sense to drop the leading `_`, not include this in the header file, and declare it `static`? (A real question, I don't remember where we landed on `static` for functions not used outside their file)

Reply:
So the reason why I stuck with this is because without the declaration in the header, I have to be careful about the order of the function definitions, whereas here I can have them in the order that is easier to read.